packages/flutter/lib/src/material/app_bar_theme.dart

```diff
@@ -173,5 +184,6 @@ class AppBarTheme with Diagnosticable {
     properties.add(DiagnosticsProperty<IconThemeData>('actionsIconTheme', actionsIconTheme, defaultValue: null));
     properties.add(DiagnosticsProperty<TextTheme>('textTheme', textTheme, defaultValue: null));
     properties.add(DiagnosticsProperty<bool>('centerTitle', centerTitle, defaultValue: null));
+    properties.add(DiagnosticsProperty<double>('titleSpacing', titleSpacing, defaultValue: null));
```

Comment:
This would be a good time to add a test to add to `AppBarTheme` for debugFillProperties, much like how [ToggleButtonsTheme does it](https://github.com/flutter/flutter/blob/c06330e18607dbb824cee3503020272d710749a8/packages/flutter/test/material/toggle_buttons_theme_test.dart#L63).